qiskit_bot.yaml

```diff
@@ -243,9 +243,6 @@ notifications:
     - "@taalexander"
     - "`@kitbarton`"
     - "`@jakelishman`"
-  "docs/guides/quickstart-org":
```

Comment:
Do we need a redirect for this page? If so, what should it go to?

Reply:
Yes - redirect quickstart-org to considerations-set-up-runtime.  Thanks @Eric-Arellano